src/transfer/jobs/openephys_job.py

```diff
@@ -1,30 +1,28 @@
 """Job that reads open ephys data, compresses, and writes it."""
-import argparse
 import subprocess
+import sys
 from pathlib import Path
 
+from botocore.session import get_session
+
+from transfer.codeocean import CodeOceanDataAssetRequests
 from transfer.compressors import EphysCompressors
 from transfer.configuration_loader import EphysJobConfigurationLoader
 from transfer.readers import EphysReaders
 from transfer.writers import EphysWriters
 
 if __name__ == "__main__":
     # Location of conf file passed in as command line arg
-    parser = argparse.ArgumentParser()
-    parser.add_argument("-c", "--conf-file-location", required=True, type=str)
-    args = parser.parse_args()
-    job_configs = EphysJobConfigurationLoader().load_configs(
-        args.conf_file_location
-    )
+    job_configs = EphysJobConfigurationLoader().load_configs(sys.argv[1:])
 
     # Extract raw data name, (e.g., openephys) and raw data path
     data_name = job_configs["data"]["name"]
-    data_src_dir = Path(job_configs["data"]["source_dir"])
-    data_dest_dir = Path(job_configs["data"]["dest_dir"])
+    data_src_dir = Path(job_configs["endpoints"]["raw_data_dir"])
+    shrunk_data_dir = Path(job_configs["endpoints"]["shrunk_data_dir"])
```

Comment:
Should this be `clipped` instead of `shrunk`?

Reply:
I think destination is fine.

Reply:
Renamed back to dest_data_dir